multimeasurements/multimeasurements.go

```diff
@@ -0,0 +1,94 @@
+// Package multimeasurements contains a helper to load a file with multiple measurements
+// and compare provided measurements against them.
+//
+// Compatible with measurements data schema v2 (see measurements.json) as well as the
+// legacy v1 schema.
+package multimeasurements
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/flashbots/cvm-reverse-proxy/internal/attestation/measurements"
+)
+
+// MultiMeasurements is holds several known measurements, and can check if a
+// given measurements matches a known one.
+type MultiMeasurements struct {
+	Measurements []MeasurementsContainer
```

Comment:
Suggestion: probably having a map<measurementID, MeasurementContaienr> would be helpful here for later operations, such as contains and direct fetching as well

Reply:
this is a 1:1 representation of the file structure. we could consider changing the internal storage structure and use a hashmap, but let's keep that for later once it's needed.